numba/tests/test_array_manipulation.py

```diff
@@ -439,6 +448,13 @@ def test_expand_dims_exceptions(self):
             cfunc(arr, 'hello')
         self.assertIn('Argument "axis" must be an integer', str(raises.exception))
```

Comment:
```suggestion
        self.assertIn('Argument "axis" must be an integer or tuple of integers', str(raises.exception))
```

Suggestion the above change the error message test to be somewhat more specific.

Reply:
Resolved in [4e4c820](https://github.com/numba/numba/pull/7604/commits/4e4c8200aae40b629579780fdcd1aa8b64096536)